test/stdlib/StringCompatibility.swift

```diff
@@ -356,5 +356,20 @@ Tests.test("LosslessStringConvertible/force unwrap/\(swift)") {
 }
 #endif
 
+#if !swift(>=4)
+Tests.test("popFirst") {
```

Comment:
Just a little comment, when the test is run in both Swift 3 and Swift 4 modes, it is quite useful to add a swift version in a test name. Like the following.

```swift
Tests.test("LosslessStringConvertible/force unwrap/\(swift)") {
```

It is not particularly important for this test right no as it only exists in Swift 3 world, but refactoring happens...